plugin-server/src/config/constants.ts

```diff
@@ -1 +1,11 @@
+import { logLevel } from 'kafkajs'
+
 export const ONE_HOUR = 60 * 60 * 1000
+
+export const KAFKAJS_LOG_LEVEL_MAPPING: Record<string, logLevel | undefined> = {
+    NOTHING: logLevel.NOTHING,
+    DEBUG: logLevel.INFO,
+    INFO: logLevel.INFO,
+    WARN: logLevel.WARN,
+    ERROR: logLevel.ERROR,
+}
```

Comment:
Kafka.js `logLevel` already is an object (an enum specifically, but uniquely for a TypeScript feature, those are actually compiled into code), so I wouldn't think this is needed?